src/components/popovers/__snapshots__/menu.test.js.snap

```diff
@@ -0,0 +1,141 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Popover component matches snapshot (JSX items) 1`] = `
+<Menu
+  trigger={
+    <button>
+      trigger
+    </button>
+  }
+>
+  <Menu>
+    <DescendantProvider
+      context={
+        Object {
+          "$$typeof": Symbol(react.context),
+          "Consumer": Object {
+            "$$typeof": Symbol(react.context),
+            "_calculateChangedBits": null,
+            "_context": [Circular],
+          },
+          "Provider": Object {
+            "$$typeof": Symbol(react.provider),
+            "_context": [Circular],
+          },
+          "_calculateChangedBits": null,
+          "_currentRenderer": Object {},
+          "_currentRenderer2": null,
+          "_currentValue": Object {
+            "descendants": Array [],
+            "registerDescendant": [Function],
+            "unregisterDescendant": [Function],
+          },
+          "_currentValue2": Object {
+            "descendants": Array [],
+            "registerDescendant": [Function],
+            "unregisterDescendant": [Function],
+          },
+          "_threadCount": 0,
+          "displayName": "MenuDescendantContext",
```

Comment:
This stuff in the snapshot isn't the greatest.

Reply:
Using a `mount` instead of `shallow` so that the `Menu` _and_ `MenuItem` components are included in the snapshot. This does a deep render which includes a lot of the stuff the library includes.

There are really three ways I can think of that this component can break due to code changes:
- className mistakenly removed
- internal nesting structure of `Menu` changes
- the library is updated (potentially with breaking api changes)

In each of these scenarios, the snapshot should break. Doing the deep render helps guarantee that will happen.